src/doc/python_scripting/functions.rst

```diff
@@ -12175,15 +12188,43 @@ WriteConfigFile
 WriteScript
 -----------
 
+**Synopsis:**
+
+::
+
+  WriteScript(f)
+
+f : file
+    The python file object that will be written to.
+
+**Description:**
+
+    WriteScript is meant to be able to write out the current state of VisIt to a Python script that can be used later to reproduce a visualization. 
+    This is like using a session file but you can take the output of WriteScript and further customize it.
+
 
 **Example:**
 
 ::
 
-  f = open('script.py', 'wt')
+  #% visit -cli
```

Comment:
WriteScript was recently modified to accept either an open file handle (as it was originally coded) or just the name of a file. So, I am proposing changes here that document both approaches...

```suggestion
  #% visit -cli
  OpenDatabase("foo.silo")
  AddPlot("Pseudocolor","dx")
  DrawPlots()
  ChangeActivePlotsVar("dy")
  WriteScript("plot_dx_and_dy.py")

or

::

    #% visit -cli  
```